spec/reek/smells/attribute_spec.rb

```diff
@@ -12,26 +11,18 @@
   it_should_behave_like 'SmellDetector'
 
   context 'with no attributes' do
-    it 'records nothing in the module' do
-      src = 'module Fred; end'
-      ctx = Reek::Core::CodeContext.new(nil, Reek::Source::SourceCode.from(src).syntax_tree)
-      expect(@detector.examine_context(ctx)).to be_empty
-    end
-
-    shared_examples_for 'no attribute found' do
-      before :each do
-        ctx = Reek::Core::CodeContext.new(nil, Reek::Source::SourceCode.from(@src).syntax_tree)
-        @smells = @detector.examine_context(ctx)
-      end
-
-      it 'records no attribute' do
-        expect(@smells.length).to eq(0)
-      end
+    it 'records nothing' do
+      expect('
+        class Fred
```

Comment:
This "Fred" thing in our specs is driving me cracy. Can we please switch that to "Klass"?